src/pages/setpassword/SetPasswordPage.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import {
+    SafeAreaView, Text, View, Image,
+} from 'react-native';
+import styles from '../../styles/styles';
+import SignInPageLayout from '../signin/SignInPageLayout';
+import SetPasswordPageProps from './SetPasswordPageProps';
+import welcomeScreenshot from '../../../assets/images/welcome-screenshot.png';
+import withWindowDimensions from '../../components/withWindowDimensions';
+import SetPasswordForm from './SetPasswordForm';
+
+const SetPasswordPage = (props) => {
+    const welcomeText = (
```

Comment:
By declaring this inside the component we will re-create it on each render. It would be better to create a new component and then use it where we are using `welcomeText`.